org._2009scape.Launcher.json

```diff
@@ -0,0 +1,66 @@
+{
+    "id": "org._2009scape.Launcher",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "2009scape-launcher",
+    "separate-locales": false,
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk11"
+    ],
+    "finish-args": [
+        "--share=network",
+        "--socket=x11",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--filesystem=home/.local/share/2009scape",
+        "--env=PATH=/usr/bin:/app/bin:/app/jre/bin"
+    ],
+    "modules": [
+        {
+            "name": "openjdk",
+            "buildsystem": "simple",
+            "build-commands" : [ "/usr/lib/sdk/openjdk11/install.sh" ]
+        },
+        {
+            "name": "2009scape",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D 2009scape-launcher-sc /app/bin/2009scape-launcher",
+                "install -Dm644 saradomin.svg /app/share/icons/hicolor/scalable/apps/org._2009scape.Launcher.svg",
+                "install -Dm644 org._2009scape.Launcher.desktop /app/share/applications/org._2009scape.Launcher.desktop",
+                "install -Dm644 org._2009scape.Launcher.appdata.xml /app/share/metainfo/org._2009scape.Launcher.appdata.xml"
+            ],
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "2009scape-launcher",
+                    "commands": [
+                          "exec 2009scape-launcher \"$@\""
+                    ]
+                },
+                {
+                    "type": "file",
+                    "url": "https://gitlab.com/2009scape/Saradomin-Launcher/-/jobs/artifacts/master/raw/Saradomin/2009scape-launcher-sc?job=pack_linux_sc",
```

Comment:
this and the other should NOT point to `master` but rather to a **specific** revision.

also this seems to be DOTnet based, but I don't see a runtime installed.

Reply:
It's Dotnet based, but it has all the dependencies included (so it works). There's a non-dependency included version, I'll see if I can get that one working with the Flatpak runtime :+1:

Reply:
I'm probably doing something wrong but I wasn't able to get the non-dependency version working with either the Mono or Dotnet extension :/. In any case, this Flatpak works on a brand new Fedora Silverblue install so it seems to be fine

Reply:
so you are packaging an x86_64 binary on aarch64.

Reply:
I did some quick research, and it seems like you can't natively run x86_64 binaries on aarch64. I'll exclude aarch64